lib/Conch/Route/Workspace.pm

```diff
@@ -102,9 +101,6 @@ sub workspace_routes {
         $with_workspace->get('/user')->to('workspace_user#list');
         # POST /workspace/:workspace_id/user
         $with_workspace->post('/user')->to('workspace_user#invite');
-
-        # POST /workspace/:workspace_id/validation_state
-        $with_workspace->get('/validation_state')->to('workspace_validation#workspace_validation_states');
```

Comment:
it's a good thing we're removing this, the docs are wrong! (GET not POST)

Reply:
doh, commit message and PR label adjusted